gbfs.md

```diff
@@ -126,19 +152,18 @@ Example:
 ```
 
 ### gbfs.json
-The following fields are all attributes within the main "data" object for this feed.
 
-Field Name              | Required    | Defines
-------------------------| ------------| ----------
-_language_              | Yes         | The language that all of the contained files will be published in. This language must match the value in the system_information file
-\-&nbsp;feeds               | Yes         | An array of all of the feeds that are published by this auto-discovery file
-\-&nbsp;name                | Yes         | Key identifying the type of feed this is (e.g. "system_information", "station_information")
-\-&nbsp;url                 | Yes         | Full URL for the feed
+Field Name              | Required    |  Type         | Defines
+------------------------| ------------| ------------  | -------------------
+_language_              | Yes         | Language code | The language that will be used throughout the rest of the files. It must match the value in the system_information file.
```

Comment:
```suggestion
_language_              | Yes         | Language | The language that will be used throughout the rest of the files. It must match the value in the system_information file.
```

since we've defined Language as a type, we don't need to add "code"